### ISSUE TITLE
go-sops pgp test fails

### ISSUE THREAD

**Issue by jvehent** [mozilla/sops#106]
``` bash
$ go test go.mozilla.org/sops/pgp
--- FAIL: TestGPG (0.26s)
    keysource_test.go:20: #1: failed on input []byte{0xc0, 0x73, 0x62, 0x4a, 0xaf, 0x39, 0x78, 0x51, 0x4e, 0xf8, 0x44, 0x3b, 0xb2, 0xa8, 0x59, 0xc7, 0x5f, 0xc3, 0xcc, 0x6a, 0xf2, 0x6d, 0x5a, 0xaa}
FAIL
FAIL    go.mozilla.org/sops/pgp 0.260s
```

**Comment by autrilla** [comment]
I can't replicate this. It passes on travis too, apparently. Have you added the PGP key to your keyring? The one on `pgp/sops_functional_tests_key.asc`.

**Comment by jvehent** [comment]
I think I've seen this bug before: go openpgp fails when the secring contains a stub to a smartcard. It's probably related.

**Comment by autrilla** [comment]
Should we maybe provide an option to call `gpg` like the Python version does instead of using go's openpgp?

**Comment by jvehent** [comment]
yes, that's a good idea.

**Comment by jvehent** [comment]
Most likely related to this: https://github.com/golang/go/issues/9312

**Comment by autrilla** [comment]
This should be fixed with #238
